spec/features/homepage_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'Homepage' do
+  # comment this out because there may be changes on how we associate the data. 
+  #
+  # context 'products exist' do
+  #   before do
+  #     @desk = create(:product)
+  #     @amazon = create(:retailer)
+  #     create(:retailer_product, product: @desk, retailer: @amazon)
+  #   end
```

Comment:
We are just going to use the produce table for now, no need for retailers. I don't think we have reliable enough data for that yet